CHANGELOG.unreleased.md

```diff
@@ -30,4 +30,5 @@
 >
 > In case of conflict, the highest (lowest in previous list) `$version` wins.
 
-- xo-web minor
+- xo-server minor
```

Comment:
Better add an entry in the CHANGELOG because the users should be aware about this functionality.